analytics/correlation_category.md

```diff
@@ -0,0 +1,35 @@
+# Correlation by Category
+
+Calculates Correlation by Category.
```

Comment:
I think we can be a little more descriptive, for example,
`Calculates correlations among groups defined by a category column.`

Reply:
Changed as suggested.